src/scripts/main.js

```diff
@@ -1,3 +1,109 @@
 'use strict';
 
-// write code here
+const TBODY = document.querySelector('.field tbody');
+const ROWS = TBODY.rows;
+
+const APPEND_ROW_BUTTON = document.querySelector('.append-row');
+const REMOVE_ROW_BUTTON = document.querySelector('.remove-row');
+const APPEND_COLUMN_BUTTON = document.querySelector('.append-column');
+const REMOVE_COLUMN_BUTTON = document.querySelector('.remove-column');
+
+const getTableSize = () => {
+  const rows = TBODY.rows;
+  const cells = rows[0].cells;
+
+  return {
+    rows: rows.length,
+    cells: cells.length,
+  };
+};
+
+const createRow = (cellsSize, rowSize) => {
```

Comment:
Code quality issue: The function `createRow` is doing too many things. It's creating a row, but also managing the state of the buttons and checking the table size. It would be better to separate these responsibilities into different functions. This would make the code cleaner and easier to understand.